app/scripts/components/common/map/utils.ts

```diff
@@ -1,17 +1,17 @@
 import axios, { Method } from 'axios';
-import { format } from 'date-fns';
+import format from 'date-fns/format';
 import { Map as MapboxMap } from 'mapbox-gl';
 import { MapRef } from 'react-map-gl';
-import { endOfDay, startOfDay } from 'date-fns';
+import startOfDay from 'date-fns/startOfDay';
+import endOfDay from 'date-fns/endOfDay';
```

Comment:
Parcel build for library was unable to find there date-fns if it wasn't imported like this... So had to switch over to doing this. Seems to be an open issue with Parcel
https://github.com/parcel-bundler/parcel/issues/9676